AdminUi/apps/admin_ui/lib/home/identities_overview/identities_filter.dart

```diff
@@ -0,0 +1,252 @@
+                _selectedClients = selectedOptions.map((item) => item.value!).toList();
+                sendFilters();
+              },
+            ),
+            Gaps.w16,
+            NumberFilter(
+              operators: operators,
+              label: 'Number of Devices',
+              onNumberSelected: (String operator, String enteredValue) {
+                _numberOfDevices = enteredValue;
+                _numberOfDevicesOperator = operator;
+
+                sendFilters();
+              },
+            ),
+            Gaps.w16,
+            DateFilter(
+              operators: operators,
+              label: 'Created At',
+              onDateSelected: (DateTime? selectedDate, String operator) {
+                setState(() {
+                  _selectedCreatedAt = selectedDate;
+                  _selectedCreatedAtOperator = operator;
+                  sendFilters();
+                });
+              },
+            ),
+            Gaps.w16,
+            DateFilter(
+              operators: operators,
+              label: 'Last Login At',
+              onDateSelected: (DateTime? selectedDate, String operator) {
+                setState(() {
+                  _selectedLastLoginAt = selectedDate;
+                  _selectedLastLoginAtOperator = operator;
+                  sendFilters();
+                });
+              },
+            ),
+            Gaps.w16,
+            NumberFilter(
+              operators: operators,
+              label: 'Datawallet Version',
+              onNumberSelected: (String operator, String enteredValue) {
+                _dataWalletVersion = enteredValue;
+                _datawalletVersionOperator = operator;
+
+                sendFilters();
+              },
+            ),
+            Gaps.w16,
+            NumberFilter(
+              operators: operators,
+              label: 'Identity Version',
+              onNumberSelected: (String operator, String enteredValue) {
+                _identityVersion = enteredValue;
+                _identityVersionOperator = operator;
+
+                sendFilters();
+              },
+            ),
+          ],
+        ),
+      ),
+    );
+  }
+
+  FilterOperator? findCorrectOperator(String operator) {
+    for (final o in comparableOperators) {
+      if (o.value == operator) {
+        return o.operator;
+      }
+    }
+    return null;
+  }
+
+  void sendFilters() {
+    filter = IdentityOverviewFilter();
+
+    if (_enteredIdentityAddress.isNotEmpty) {
+      filter = filter.copyWith(address: _enteredIdentityAddress);
+    }
+    if (_selectedTiers.isNotEmpty) {
+      filter = filter.copyWith(tiers: _selectedTiers);
+    }
+    if (_selectedClients.isNotEmpty) {
+      filter = filter.copyWith(clients: _selectedClients);
+    }
+
+    if (_selectedCreatedAt != null) {
+      final createdAtValue = FilterOperatorValue(findCorrectOperator(_selectedCreatedAtOperator)!, _selectedCreatedAt.toString().substring(0, 10));
```

Comment:
Because the date that i am selecting from a date picker has a time of 00:00:00 but the time on an actual Date from a OData has a different time and so it will not be the same no matter what.
![image](https://github.com/nmshd/backbone/assets/37671301/ca7965bf-64de-4cb2-8bac-a7c262fb845e)

Reply:
Ah, please do that using dateformat.

Reply:
Changes made.